docs/src/makie.md

```diff
@@ -0,0 +1,66 @@
+# Makie.jl plots
+
+Compared to Gadfly.jl and StatsPlots.jl, Makie.jl is the most flexible plotting library that you can use.
+On this page is an example function for plotting with Makie.jl which you can use directly or further tweak to your needs.
+
+```@example makie
+using CairoMakie
+using DataFrames
+using MCMCChains
+
+chns = Chains(randn(300, 5, 3), [:A, :B, :C, :D, :E])
+```
+
+```@example makie
+function plot_chains(chns; density_func=density!)
```

Comment:
Defining a function that is only called once seems unnecessarily complicated for an example. Just write it as a single script instead? And if possible, maybe split it into different steps, explain them and show the intermediate results similar to the MakieLayout tutorial?